src/e3/anod/driver.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 from functools import wraps
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Optional
```

Comment:
Do not use Optional but just | None

Reply:
Ok. It's just that Rust have that `Option` type which is widely used, so I thought it was a good idea to use :) Will do.